src/nodiff.js

```diff
@@ -1,3 +1,126 @@
-export default async function nodiff() {
-  // TODO(dabrady) Write the code that does the thing.
+import { info, isDebug, setFailed } from '@actions/core';
+import { exec } from '@actions/exec';
+import {
+  hydrateTemplateString,
+  leaveComment,
+  requestChanges,
+  requestReviewers,
+} from './helpers';
+
+const FAILURE_MESSAGE = `Meaningless changes have been made to:\n`;
```

Comment:
```suggestion
const FAILURE_MESSAGE = `Meaningless or whitespace-only changes have been made to:\n`;
```

Reply:
I don't want to make this precise change, because this action is intentionally using the broad term `meaningless`, to allow room for expansion. But I agree, the folks reading this message won't necessarily know that, so I should come up with a better phrasing here (or parameterize this message).

Reply:
I decided to basically go with your suggestion, and added a TODO to improve it if the scope of this Action is ever broadened.